examples/long-short.py

```diff
@@ -11,328 +12,275 @@
+                    incomplete.append(stock)
+                else:
+                    executed.append(stock)
+        return [executed, incomplete]
+
+    # Submit an order if quantity is above 0.
+    def submit_order(self, qty, stock, side) -> bool:
+        if qty > 0:
+            try:
+                self.alpaca.submit_order(stock, qty, side, "market", "day")
+                print(f"Market order of | {qty} {stock} {side} | completed.")
+                return True
+            except Exception as e:
+                print(f"Order of | {qty} {stock} {side} | did not go through. Exception: {e}")
+                return False
         else:
-          executed.append(stock)
-        respSO.clear()
-    resp.append([executed, incomplete])
-
-  # Submit an order if quantity is above 0.
-  def submitOrder(self, qty, stock, side, resp):
-    if(qty > 0):
-      try:
-        self.alpaca.submit_order(stock, qty, side, "market", "day")
-        print("Market order of | " + str(qty) + " " + stock + " " + side + " | completed.")
-        resp.append(True)
-      except:
-        print("Order of | " + str(qty) + " " + stock + " " + side + " | did not go through.")
-        resp.append(False)
-    else:
-      print("Quantity is 0, order of | " + str(qty) + " " + stock + " " + side + " | not completed.")
-      resp.append(True)
-
-  # Get percent changes of the stock prices over the past 10 minutes.
-  def getPercentChanges(self):
-    length = 10
-    for i, stock in enumerate(self.allStocks):
-      bars = self.alpaca.get_bars(stock[0], TimeFrame.Minute,
-                                  pd.Timestamp('now').date(),
-                                  pd.Timestamp('now').date(), limit=length,
-                                  adjustment='raw')
-      self.allStocks[i][1] = (bars[stock[0]][len(bars[stock[0]]) - 1].c - bars[stock[0]][0].o) / bars[stock[0]][0].o
-
-  # Mechanism used to rank the stocks, the basis of the Long-Short Equity Strategy.
-  def rank(self):
-    # Ranks all stocks by percent change over the past 10 minutes (higher is better).
-    tGetPC = threading.Thread(target=self.getPercentChanges)
-    tGetPC.start()
-    tGetPC.join()
-
-    # Sort the stocks in place by the percent change field (marked by pc).
-    self.allStocks.sort(key=lambda x: x[1])
+            print("Quantity is 0, order of | {qty} {stock} {side} | not completed.")
+            return True
+
+    # Get percent changes of the stock prices over the past 10 minutes.
+    def get_percent_changes(self) -> None:
+        length = 10
+        for i, stock in enumerate(self.all_stocks):
+            bars = self.alpaca.get_bars(stock[0], TimeFrame.Minute,
+                                        datetime.date.today().isoformat(),
+                                        datetime.date.today().isoformat(), limit=length,
+                                        adjustment='raw')
+            self.all_stocks[i][1] = (bars[stock[0]][len(bars[stock[0]]) - 1].c -
```

Comment:
[minor] You can simply use -1
```suggestion
            self.all_stocks[i][1] = (bars[stock[0]][-1].c -
```